### ISSUE TITLE
[FEATURE]: worktime balance in navbar

### ISSUE THREAD

**Issue by gianklug** [adfinis/timed#468]
### Is your feature request related to a problem? Please describe.

To check their current worktime balance, a user always has to open the profile page. It would be nice to have it directly visible.

### Describe the solution you'd like

An additional navbar entry showing the current worktime balance. Example:

![image](https://github.com/user-attachments/assets/de1ab9c8-813d-498f-8391-56548be83547)

### Describe alternatives you've considered

Maybe this isn't as good in terms of privacy; e.g. other people looking at one's screen with an open timed window. 

An alternative would be to add a popover when hovering on the user icon; this would still eliminate the need to open an additional page.

### Required

- [X] I have checked past issues to see if this feature has already been requested.

**Comment by derrabauke** [comment]
*imho* the top right bar is quite full already (Reviews, Rejected Reports, etc). But we could actually "hide" that feature in a *obvious* way. For example within the clock. 

- **3h hours minus**
![3h_minus](https://github.com/user-attachments/assets/e743fe71-d866-4335-97fe-c7e660ec7c4e)

- **6h plus**
![6h_plus](https://github.com/user-attachments/assets/d20ab88f-018e-44ac-9d57-669ea213a72a)

Everything above or below 12h is *capped*, but could eventually influence the saturation of the colored background. Colors must adhere to the theme - I just picked random colors for a quick sketch.

**Comment by derrabauke** [comment]
And giving the user a way to turn that *on/off* would be nice as well.

**Comment by gianklug** [comment]
Fair point; I don't usually have reviews or rejected reports in my position; didn't think of that.

I think making the end-user experience more customizable would be great in general, e.g. changing to a dark theme (in the future), setting the default view, etc.

The clock approach looks nice, although with 0 overtime hours, it'll be red for most of the time. 


Maybe something like this would also be a viable option :thinking: 
![image](https://github.com/user-attachments/assets/6740fd0e-6b87-453b-a5cf-b4d9d58f6e5f)

**Comment by derrabauke** [comment]
I've spend some personal free time to implement this. 

Here are my proposals, what you say @gianklug @trowik @c0rydoras?

**indebted**
![Image](https://github.com/user-attachments/assets/05044b25-fb85-48c9-8e6c-222927b1dc4b)

**quite neutral**
![Image](https://github.com/user-attachments/assets/fa094b6d-fd68-4342-9863-a32bb4aa781e)

**lots of overtime**
![Image](https://github.com/user-attachments/assets/dcc227d4-6444-46c9-a1e0-2f99ff4479d5)

Further I would like to add a *settings* view where users could adjust their personal preferences regarding theming, displaying of this feature and maybe others in the future. 🤷
